WalletWasabi.Fluent/ViewModels/SearchBar/SearchBarViewModel.cs

```diff
@@ -28,11 +28,10 @@ public SearchBarViewModel(IObservable<IChangeSet<ISearchItem, ComposedKey>> item
 			.ObserveOn(RxApp.MainThreadScheduler)
 			.Subscribe();
 
-		HasResults = itemsObservable
-			.Count()
-			.Select(i => i > 0)
-			.Replay(1)
-			.ReplayLastActive();
+		_hasResults = itemsObservable
+			.ToCollection()
+			.Select(x => x.Any())
+			.ToProperty(this, x => x.HasResults);
```

Comment:
HasResults as `AutoNotify` property and using BindTo() here didn't work?
I would just avoid OAPH if possible to keep the consistency.